File: month-exams-parent/month-exams-interface/src/main/java/com/month/exams/service/GoodsService.java
package com.month.exams.service;

import java.util.List;

import com.month.exams.pojo.TGoods;

public interface GoodsService {

	public List<TGoods> getList();

	public void save(TGoods tGoods);

	public void dwon(int id);

	public void uo(int id);

	public TGoods findId(int id);
}

File: month-exams-parent/month-exams-web/src/main/webapp/js/service/goodsService.js
/**
 * 
 */
app.service('goodsService',function($http){
	this.getList=function(){
		return $http.get('../goods/getList.do');
	}
	
	this.save=function(entity){
		return $http.post('../goods/save.do',entity);
	}
	
	this.up=function(id){
		return $http.get('../goods/up.do?id='+id);
	}
	
	this.dwon=function(id){
		return $http.get('../goods/dwon.do?id='+id);
	}
})